amd_openvx_extensions/amd_winml/include/vx_ext_winml.h

```diff
@@ -63,66 +78,44 @@ extern "C"
 	 * \ingroup group_amd_winml
 	 * @note Kernel Name: com.winml.onnx_to_mivisionx
 	 * \param [in] graph The reference to the graph.
-	 * \param [in] input_1 The ONNX Model Location in vx_scalar.
-	 * \param [in] input_2 The ONNX Model Input Tensor Name in vx_scalar.
-	 * \param [in] input_3 The ONNX Model Output Tensor Name in vx_scalar.
-	 * \param [in] input_4 The Input Tensor in <tt>\ref VX_FLOAT32</tt> format.
-	 * \param [in] input_5 The setup Array for each model in <tt>\ref VX_TYPE_SIZE</tt> format.
-	 * \param [out] output The output Tensor in <tt>\ref VX_FLOAT32</tt> format.
-	 * \param [in] input_6 WinML Deploy Device Kind in vx_scalar [optional] (default: 0).
+	 * \param [in] modelLocation The ONNX Model Location in vx_scalar.
+	 * \param [in] inputTensorName The ONNX Model Input Tensor Name in vx_scalar.
+	 * \param [in] outputTensorName The ONNX Model Output Tensor Name in vx_scalar.
+	 * \param [in] inputTensor The Input Tensor.
+	 * \param [in] setupArray The setup Array for each model in <tt>\ref VX_TYPE_SIZE</tt> format.
+	 * \param [out] outputTensor The output Tensor.
+	 * \param [in] deviceKind WinML Deploy Device Kind in vx_scalar [optional] (default: 0).
 	 * \return <tt>\ref vx_node</tt>.
 	 * \retval vx_node A node reference. Any possible errors preventing a successful creation should be checked using <tt>\ref vxGetStatus</tt>*/
-	extern "C" SHARED_PUBLIC vx_node VX_API_CALL vxExtWinMLNode_OnnxToMivisionX(
-		vx_graph graph,
-		vx_scalar modelLocation,
-		vx_scalar inputTensorName,
-		vx_scalar outputTensorName,
-		vx_tensor inputTensor,
-		vx_array setupArray,
-		vx_tensor outputTensor,
-		vx_scalar deviceKind);
+	extern "C" SHARED_PUBLIC vx_node VX_API_CALL vxExtWinMLNode_OnnxToMivisionX(vx_graph graph, vx_scalar modelLocation, vx_scalar inputTensorName, vx_scalar outputTensorName, vx_tensor inputTensor, vx_array setupArray, vx_tensor outputTensor, vx_scalar deviceKind);
 
 	/*! \brief [Graph] Creates a WinML convert image to tensor node.
 	 * \ingroup group_amd_winml
 	 * @note Kernel Name: com.winml.convert_image_to_tensor
 	 * \param [in] graph The reference to the graph.
-	 * \param [in] input_1 input image vx_image.
-	 * \param [out]  The output Tensor in <tt>\ref VX_FLOAT32</tt> format.
-	 * \param [in] input_2 a in vx_scalar.
-	 * \param [in] input_3 b in vx_scalar.
-	 * \param [in] input_4 reverse channel order in vx_scalar.
+	 * \param [in] input input image vx_image.
+	 * \param [out] output The output Tensor.
+	 * \param [in] a a in vx_scalar.
```

Comment:
Typo here

Reply:
Not a typo, changed it for better wording